src/templates/antora/util.ts

```diff
@@ -0,0 +1,252 @@
+import * as fs from 'fs';
+import * as Handlebars from 'handlebars';
+import * as YAML from 'js-yaml';
+import * as path from 'path';
+
+import { ExportStructure } from '../../lib/exporter';
+import { Type } from '../../lib/type';
+
+interface NavFile {
+  readonly title: string;
+  readonly path: string;
+  readonly pages?: NavFile[];
+}
+
+export interface PageOutput {
+  readonly type: 'adoc' | 'json' | 'yml';
+  readonly filename: string;
+  readonly content: string;
+}
+
+const BASE_PATH: string = process.env.BASE_PATH || '/_data/antora';
+
+const namespaceDescriptions = {
+  'tinymce': 'Global APIs for working with the editor.',
+  'tinymce.dom': 'APIs for working with the DOM from within the editor.',
+  'tinymce.editor.ui': 'APIs for registering User Interface components.',
+  'tinymce.geom': 'Various rectangle APIs.',
+  'tinymce.html': 'APIs for working with HTML within the editor.',
+  'tinymce.util': 'Browser related APIs.'
+};
+
+/**
+ * [compileTemplate description]
+ * @param  {[type]} filePath [description]
+ * @return {[type]}          [description]
+ */
+const compileTemplate = (filePath: string): HandlebarsTemplateDelegate => {
+  return Handlebars.compile(fs.readFileSync(path.join(__dirname, filePath)).toString());
+};
+
+const getNamespaceFromFullName = (fullName: string): string =>
+  fullName === 'tinymce' ? fullName : fullName.split('.').slice(0, -1).join('.');
+
+const getApiFromFullName = (fullName: string): string =>
+  fullName === 'tinymce' ? fullName : fullName.split('.').slice(1).join('.');
+
+const getNameFromFullName = (fullName: string): string =>
+  fullName === 'tinymce' ? fullName : fullName.split('.').slice(-1).join('.');
+
+const getTitleFromFullName = (fullName: string): string =>
+  fullName.split('.').slice(-1).join('');
+
+const navLine = (name: string, level: number): string =>
+  '*'.repeat(level) + ' ' + name + '\n';
+
+const generateNavXref = (basePath: string, filename: string, title: string): string =>
+  'xref:' + basePath + '/' + filename + '[' + title + ']';
+
+const generateXref = (name: string, structure: ExportStructure): string => {
+  const title = getTitleFromFullName(name);
+  const fileName = name.toLowerCase() === 'tinymce' ? getRootPath(structure) : name.toLowerCase();
+  switch (structure) {
+    case 'legacy':
+      return generateNavXref('api/' + getNamespaceFromFullName(name.toLowerCase()), fileName + '.adoc', title);
+
+    case 'default':
+      return generateNavXref('apis', fileName + '.adoc', title);
+  }
+};
+
+const generateTypeXref = (type: string, structure: ExportStructure): string => {
+  return type.includes('tinymce', 0) ? generateXref(type, structure) : type;
+};
+
+const getJsonFilePath = (type: string, fullName: string): string =>
+  (BASE_PATH + '/api/json/' + type + '_' + fullName.replace(/\./g, '_') + '.json').toLowerCase();
```

Comment:
This path is incorrect and introduces a bug because of that... atm the JSON content is being transferred to Antora which we don't want. If you look at the old version this specific path should be hardcoded to `_data/api/json` instead of using the `BASE_PATH` since it's not Antora content.

- [ ] Fix bug with JSON output being placed in the wrong location

Reply:
Fixed